src/collections/_documentation/guides/getting-started/index.md

```diff
@@ -0,0 +1,125 @@
+
+![Create New Team]({% asset guides/getting-started/03.png @path %})
+
+Click on a team name to open the team settings to manage its associated members and projects.
+Under the `Members` tab add existing members to your team by clicking on `Add Member > [Member Name]`.
+
+![Add Members]({% asset guides/getting-started/04.png @path %})
+
+You can also invite multiple new (external) members to join your organization and team by clicking on `Add Member > Invite Member`
+
+![Invite Members]({% asset guides/getting-started/05.png @path %})
+
+For more information see: [Organization Management](https://docs.sentry.io/accounts/membership/)
+
+## 4. Create Projects
+
+To start monitoring errors in your app with Sentry you'll need to initialize the SDK with a DSN key. To obtain a key, simply create a new Sentry project by going to `Settings > Projects > + Create Project`. Give the project a name and assign the responsible [Team/s](#3-set-up-teams). Now, any error that occurs in your code (wherever it may be deployed and running) will be associated with that specific project.
+
+![Create a Project]({% asset guides/getting-started/08.png @path %})
+
+### > **What's in a Project?**
+
+A Project is a logical entity that connects between the errors in your application (or a part of it) to the team members assigned to that project. The **Project Settings** configuration determines
+
+- Which errors should be ingested into your Sentry account and which should be dropped, through [Inbound Filters](https://docs.sentry.io/accounts/quotas/#inbound-data-filters)
+- Who to notify, about what error, and how, using [Alert Rules](https://docs.sentry.io/workflow/notifications/alerts/)
+- Which errors should be automatically assigned to which member or team through [Issue Owners](https://docs.sentry.io/workflow/issue-owners/)
+- Custom rules to fine-tune [Event Grouping](https://docs.sentry.io/data-management/event-grouping/) into issues.
+
+Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, developers are looking at information that is relevant to their work, and see the errors that they might need to jump on.
+
+![My Issue Stream]({% asset guides/getting-started/09.png @path %})
+
+### > **How Many Projects Should I Create?**
+
+You could theoretically pour all your errors into a single project. The Sentry UI will not limit you. However, considering the above ([What's in a Project?](#-whats-in-a-project)), setting up multiple projects to reflect your application landscape with finer granularity, will go a long way with regards to visibility, actionability, and productivity.
+
+Here are some points to consider:
+
+- If your app source code is managed in multiple repositories, create a separate project for each repo.
```

Comment:
Can we add some formatting / bolding here to each of these entries. i.e. **several micro-service** -> **split**

Up to you (: